statistics/testdata/stats_suite_out.json

```diff
@@ -7,6 +7,417 @@
+      ],
+      [
+        "IndexLookUp_10 1.00 root  ",
+        "├─IndexRangeScan_8(Build) 1.00 cop[tikv] table:tdecimal, index:multi(b, c) range:[1.00000000000000000000,1.00000000000000000000], keep order:false",
+        "└─TableRowIDScan_9(Probe) 1.00 cop[tikv] table:tdecimal keep order:false"
+      ],
+      [
+        "IndexLookUp_10 1.00 root  ",
+        "├─IndexRangeScan_8(Build) 1.00 cop[tikv] table:tdecimal, index:multi(b, c) range:[4.00000000000000000000,4.00000000000000000000], keep order:false",
+        "└─TableRowIDScan_9(Probe) 1.00 cop[tikv] table:tdecimal keep order:false"
+      ],
+      [
+        "IndexLookUp_10 1.00 root  ",
+        "├─IndexRangeScan_8(Build) 1.00 cop[tikv] table:tdecimal, index:multi(b, c) range:[8.00000000000000000000,8.00000000000000000000], keep order:false",
+        "└─TableRowIDScan_9(Probe) 1.00 cop[tikv] table:tdecimal keep order:false"
+      ],
+      [
+        "TableReader_7 1.00 root  data:Selection_6",
+        "└─Selection_6 1.00 cop[tikv]  eq(test.tstring.b, \"1\")",
+        "  └─TableFullScan_5 8.00 cop[tikv] table:tstring keep order:false"
+      ],
+      [
+        "TableReader_7 1.00 root  data:Selection_6",
+        "└─Selection_6 1.00 cop[tikv]  eq(test.tstring.b, \"4\")",
+        "  └─TableFullScan_5 8.00 cop[tikv] table:tstring keep order:false"
+      ],
+      [
+        "TableReader_7 1.00 root  data:Selection_6",
+        "└─Selection_6 1.00 cop[tikv]  eq(test.tstring.b, \"8\")",
+        "  └─TableFullScan_5 8.00 cop[tikv] table:tstring keep order:false"
+      ],
+      [
+        "TableReader_7 1.00 root  data:Selection_6",
+        "└─Selection_6 1.00 cop[tikv]  eq(test.tdatetime.b, 2001-01-01 00:00:00.000000)",
+        "  └─TableFullScan_5 4.00 cop[tikv] table:tdatetime keep order:false"
+      ],
+      [
+        "TableReader_7 1.00 root  data:Selection_6",
+        "└─Selection_6 1.00 cop[tikv]  eq(test.tdatetime.b, 2001-01-02 00:00:00.000000)",
+        "  └─TableFullScan_5 4.00 cop[tikv] table:tdatetime keep order:false"
+      ],
+      [
+        "TableReader_7 1.00 root  data:Selection_6",
+        "└─Selection_6 1.00 cop[tikv]  eq(test.tdatetime.b, 2001-01-04 00:00:00.000000)",
+        "  └─TableFullScan_5 4.00 cop[tikv] table:tdatetime keep order:false"
+      ],
+      [
+        "TableReader_6 4.00 root  data:TableRangeScan_5",
+        "└─TableRangeScan_5 4.00 cop[tikv] table:ct1 range:[\"1\",\"4\"], keep order:false"
+      ],
+      [
+        "TableReader_6 2.00 root  data:TableRangeScan_5",
+        "└─TableRangeScan_5 2.00 cop[tikv] table:ct1 range:[\"4\",\"6\"], keep order:false"
+      ],
+      [
+        "TableReader_6 3.00 root  data:TableRangeScan_5",
+        "└─TableRangeScan_5 3.00 cop[tikv] table:ct1 range:[\"6\",\"8\"], keep order:false"
+      ],
+      [
+        "TableReader_6 0.00 root  data:TableRangeScan_5",
+        "└─TableRangeScan_5 0.00 cop[tikv] table:ct2 range:[1 1,1 8], keep order:false"
```

Comment:
This should be `1.00`.

Reply:
This case almost has the same reason with the previous one.
It is calculated in `Hist.BetweenRowCount`: 0 = lessCount([1, 9]) - lessCount([1, 1]);

The first bucket's range is from [1, 1] to [4, 4], so lessCount([1, 1]) = 0;

When calculating lessCount([1, 9]), we have to calculate bucket[0].fraction([1, 9]).
Since [1, 9] has two columns, it will be regarded as string([]byte) when calculating fraction, which leads to 0.